controllers/ibmblockcsi_controller.go

```diff
@@ -308,6 +317,14 @@ func (r *IBMBlockCSIReconciler) updateStatus(instance *ibmblockcsi.IBMBlockCSI,
 		return err
 	}
 
+	callHomeCronJob, err := r.getCallHomeCronJob(instance)
```

Comment:
what if we fail to get the cronjob here?

Reply:
done

Reply:
I think we need to distinguish between idempotency and other errors

Reply:
done